node/module_all.ts

```diff
@@ -69,7 +75,13 @@ export default {
   timers,
   "timers/promises": timersPromises,
   tty,
+  tls: {},
   url,
+  "worker_threads": {},
   util,
+  vm: {
+    Script,
+  },
```

Comment:
Open a separate PR for this and make methods on `Script` throw not-implemented error